content/en/security/application_security/enabling/compatibility/java.md

```diff
@@ -6,45 +6,131 @@ type: multi-code-lang
+| 18 to 19     | Windows (x86, x86-64)<br>Linux (x86, x86-64, arm64)<br>Mac (x86, x86-64, arm64) | [Beta](#levels-of-support)               | Latest         |
+| 8 to 17      | Windows (x86, x86-64)<br>Linux (x86, x86-64)<br>Mac (x86, x86-64)               | [GA](#levels-of-support)                   | Latest         |
+| 8 to 17      | Linux (arm64)<br>Mac (arm64)                                                    | [Beta](#levels-of-support)               | Latest         |
+| 7            | Windows (x86, x86-64)<br>Linux (x86, x86-64)<br>Mac (x86, x86-64)               | [Maintenance](#levels-of-support) | v0             |
+| 7            | Linux (arm64)<br>Mac (arm64)                                                    | [End-of-life](#levels-of-support)         | v0             |
 
 Datadog does not officially support any early-access versions of Java.
 
 You can monitor application security for Java apps running in Docker, Kubernetes, AWS ECS, and AWS Fargate.
 
-### Supported frameworks
+## Integrations
+
+### Levels of framework support
+
+| **Level**                                              | **Support provided**                                                                                                                                |
+|--------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| <span id="support-unsupported">Not Supported</span>      |  No implementation. Contact [Datadog support][2] for special requests.                                                                              |
+| <span id="full-support">Supported</span>                    |  All capabilities work as intended. ASM can detect threats, block requests and report vulnerabilities. |                                                                                                                                   
+
+### Web framework compatibility
+
+- Attacker source HTTP request details
+- Tags for the HTTP request (status code, method, etc)
+- Distributed Tracing to see attack flows through your applications
 
-| Framework Web Server    | Minimum Framework Version   |
-| ----------------------- | --------------------------- |
-| Servlet Compatible      | 2.3+, 3.0+                  |
-| Spring                  | 3.1                         |
+##### ASM Capability Notes
+- Open Source Vulnerability Detections is supported on all libraries
+
+
+
+| Framework                  | Versions   | Threat Detection supported? | Threat Protection supported? | Custom Code Vulnerability Detection supported? |
+| ----------------------- | ---------- | --------------- | ---------------------------------------------- | ---------------------------------------------- |
+| Grizzly                 | 2.0+       |  {{< X >}} |  {{< X >}} |  {{< X >}} |
```

Comment:
@sourabhkatti I have also removed all the "supported" verbiage to the shortcode `{{< X >}}` as this renders as a bold checkmark, and removed the verbiage "not supported" to just blank.